command/operator_raft_list.go

```diff
@@ -0,0 +1,81 @@
+package command
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/command/base"
+	"github.com/ryanuber/columnize"
+)
+
+type OperatorRaftListCommand struct {
+	base.Command
+}
+
+func (c *OperatorRaftListCommand) Help() string {
+	helpText := `
+Usage: consul operator raft list-peers [options]
+
+Displays the current Raft peer configuration.
+
+` + c.Command.Help()
+
+	return strings.TrimSpace(helpText)
+}
+
+func (c *OperatorRaftListCommand) Synopsis() string {
+	return "Display the current Raft peer configuration"
+}
+
+func (c *OperatorRaftListCommand) Run(args []string) int {
+	c.Command.NewFlagSet(c)
+
+	if err := c.Command.Parse(args); err != nil {
+		if err == flag.ErrHelp {
+			return 0
+		}
+		c.Ui.Error(fmt.Sprintf("Failed to parse args: %v", err))
+		return 1
+	}
+
+	// Set up a client.
+	client, err := c.Command.HTTPClient()
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error initializing client: %s", err))
+		return 1
+	}
+
+	// Fetch the current configuration.
+	result, err := raftListPeers(client.Operator(), c.Command.HTTPStale())
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error getting peers: %v", err))
+	}
+	c.Ui.Output(result)
+
+	return 0
+}
+
+func raftListPeers(operator *api.Operator, stale bool) (string, error) {
+	q := &api.QueryOptions{
```

Comment:
the api client doesn't have any way to set a config setting for stale when initializing it, so this is the only way currently